packages/datadog-plugin-aws-sdk/src/services/stepfunctions.js

```diff
@@ -0,0 +1,61 @@
+'use strict'
+const log = require('../../../dd-trace/src/log')
+const BaseAwsSdkPlugin = require('../base')
+
+class Stepfunctions extends BaseAwsSdkPlugin {
+  static get id () { return 'sfn' }
+
+  // "StartExecutionInput": {
```

Comment:
Let's not leave commented code unless there is a good reason documented in a comment.

Reply:
👍 This is the shape of StartExecutionInput from https://github.com/aws/aws-sdk-js/blob/master/apis/states-2016-11-23.normal.json . I added a comment to this effect.